.github/actions/e2e-atomic-csp/action.yml

```
@@ -5,8 +5,8 @@ runs:
   steps:
     - name: Complementary setup
       run: |
-        npm ci
-        npx turbo build --filter=@coveo/atomic
+        pnpm install
```

`--frozen-lockfile` ? Seems it's the equivalent of `npm ci` ?

```suggestion
        pnpm install --frozen-lockfile
```